controlplane/kubeadm/api/v1alpha3/kubeadm_control_plane_webhook_test.go

```diff
@@ -278,6 +358,120 @@ func TestKubeadmControlPlaneValidateUpdate(t *testing.T) {
 			before:    beforeExternalEtcdCluster,
 			kcp:       scaleToEvenExternalEtcdCluster,
 		},
+		{
+			name:      "should succeed when making a change to the local etcd image tag",
+			expectErr: false,
+			before:    before,
+			kcp:       etcdLocalImageTag,
+		},
+		{
+			name:      "should fail when making a change to the cluster config's networking struct",
+			expectErr: true,
+			before:    before,
+			kcp:       networking,
+		},
+		{
+			name:      "should fail when making a change to the cluster config's kubernetes version",
+			expectErr: true,
+			before:    before,
+			kcp:       kubernetesVersion,
+		},
+		{
+			name:      "should fail when making a change to the cluster config's controlPlaneEndpoint",
+			expectErr: true,
+			before:    before,
+			kcp:       controlPlaneEndpoint,
+		},
+		{
+			name:      "should fail when making a change to the cluster config's apiServer",
+			expectErr: true,
+			before:    before,
+			kcp:       apiServer,
+		},
+		{
+			name:      "should fail when making a change to the cluster config's controllerManager",
+			expectErr: true,
+			before:    before,
+			kcp:       controllerManager,
+		},
+		{
+			name:      "should fail when making a change to the cluster config's scheduler",
+			expectErr: true,
+			before:    before,
+			kcp:       scheduler,
+		},
+		{
+			name:      "should fail when making a change to the cluster config's dns",
```

Comment:
We'll need this to be allowed as part of #2574, either here or as part of @wfernandes's PR, just FYI